lib/views/general/_frontpage_requests_list.html.erb

```diff
@@ -1,15 +1,17 @@
 <%- @request_events, @request_events_all_successful = InfoRequest.recent_requests %>
-<div id="examples_1">
+<div id="examples_1" class="bodies-list">
```

Comment:
FWIW we use [`latest-requests` in Alaveteli core](https://github.com/mysociety/alaveteli/blob/develop/app/views/general/_frontpage_requests_list.html.erb#L1).